common/modules/disable-add-liq/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![no_std]
+
+multiversx_sc::imports!();
+
+pub const ADD_LIQ_ENABLED: bool = true;
+pub const ADD_LIQ_DISABLED: bool = false;
+
+#[multiversx_sc::module]
+pub trait DisableAddLiqModule {
+    #[only_owner]
+    #[endpoint(enableAddLiq)]
+    fn enable_add_liq(&self) {
+        self.add_liq_enabled().set(ADD_LIQ_ENABLED);
+    }
+
+    #[only_owner]
+    #[endpoint(disableAddLiq)]
+    fn disable_add_liq(&self) {
+        self.add_liq_enabled().set(ADD_LIQ_DISABLED);
+    }
+
+    fn require_add_liq_enabled(&self) {
```

Comment:
What if this would act in a reverse way? Not add liquidity enabled, but add liquidity paused. So you have to write in storage whenever you want to pause the adding of liquidity. In other words, we would modify only this module, not the implementation in other contracts. We could keep require_add_liq_enabled as is, but check the storage has something instead of being empty.
This would be helpful when upgrading the contracts, so that the owner doesn't need to call the extra allow function.

Reply:
...I think it was you who told me to do it this way, and now you want it the other way again?